srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'epson-inkjet-printer-escpr2'
+pkgname=epson-inkjet-printer-escpr2
+version=1.1.15
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
+create_wrksrc=no
+build_wrksrc=${pkgname}-${version}
+build_style=configure
+configure_script=./configure
+configure_args="--prefix=/usr --with-cupsfilterdir=/usr/lib/cups/filter --with-cupsppddir=/usr/share/ppd"
```

Comment:
Remove `--prefix=/usr`

Reply:
Done next time I amend it , that is gone

Reply:
--prefix=/usr , is necessary.

If I remove that /usr/local will be used instead , and that is not allowed.

Reply:
No, i don't think it's nessecary, since over in https://github.com/void-linux/void-packages/blob/master/common/environment/configure/gnu-configure-args.sh#L7 it sets all that, @Chocimier is this correct?

Reply:
Change to `build_style=gnu-configure`

Reply:
I have changed the build style to gnu-configure and remove the --prefix=/usr